web/src/components/spinner-button.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import '../../css/spinner-button.css';
+
+ export default class SpinnerButton extends React.Component {
```

Comment:
React usually uses classes when there is logic around the component lifecycle apart from rendering. For example there is a `componentDidMount`. You can read more about lifecycles here: https://reactjs.org/docs/state-and-lifecycle.html

If none of these lifecycles applies, you can also use "stateless functional components" (https://hackernoon.com/react-stateless-functional-components-nine-wins-you-might-have-overlooked-997b0d933dbc). This makes the component more readable as it's shorter and straightforward.

```
import React from 'react';

import '../../css/spinner-button.css';

const SpinnerButton = () => {
  return (
    <button className="spinnerButton spinning" disabled="disabled">Submit</button>
  );
};

export default SpinnerButton;
```

I know that we don't consistently apply this through the whole code, but I think that might be a good start here :)

Reply:
Thank you for explaining me this :) I actually thought about doing it this way, but I some in most files in was done in the other way so I chose the other one to be consistent with the project. I agree this way is shorter and easier to understand.